components/widgets/resources/src/main/resources/resources/uicomponents/widgets/geneValidation.js

```diff
@@ -20,28 +20,33 @@ var XWiki = (function(XWiki) {
         this.input.__validation.add(this.validate.bind(this));
     },
     check : function() {
-      if (this.input.value != this.value) {
+      if (this.input.value.blank() || this.input.value != this.value) {
         this.value = this.input.value;
         this.state = 'CHECKING';
         var el = this.input;
-        var genesymbols = [];
-        $$('.gene.col-label.gene-input-label').each( function (item) {
-          if (item.next() != el)
-            genesymbols.push(item.textContent || item.innerText);
-        });
-        if (genesymbols.indexOf(el.value) > -1) {
+        if (el.value.blank()) {
           this.invalid();
-        } else {
-          this.available();
+        } else if (this.input.className.include('gene-name')){
+          var genesymbols = [];
+          $$('.gene.col-label.gene-input-label').each( function (item) {
+            if (item.next() != el) {
+              genesymbols.push(item.textContent || item.innerText);
+            }
+          });
+          if (genesymbols.indexOf(el.value) > -1) {
+            this.invalid();
+          } else {
+            this.available();
+          }
         }
         this.responded();
       }
     },
     validate : function(value) {
-      if (this.state == 'DONE' &&
-          this.value == value &&
-          !this.valid) {
-        Validate.fail("This gene has already been entered.");
+      if ((this.state == 'DONE' && this.value == value && !this.valid) ||
+          (this.state == 'NEW' && value.blank())) {
+        var message = (value.blank()) ? "Input is blank." : "This gene has already been entered.";
```

Comment:
`Input is blank` sounds unfriendly. `Please enter a value` is better, but `Please enter a [gene name|variant]` would be best.